mod/wikirate/set/type/image/wikirate_image.rb

```diff
@@ -0,0 +1,8 @@
+view :missing do |args|
+  if home_view = args[:home_view] and home_view == :source
+    # to get the wikirate icon
+    Card["*logo"].format.render_source args.merge({:size=>"large"})
```

Comment:
I am not sure whether this approach is fine or not. 

I need to get the wikirate logo url while that image is missing for the facebook meta